src/components/LibraryInfo/index.js

```diff
@@ -12,14 +12,18 @@ governing permissions and limitations under the License.
 
 import libraryVersion from "../../constants/libraryVersion";
 
-const createLibraryInfo = () => {
+const createLibraryInfo = ({ config, componentRegistry }) => {
+  const coreCommandsArr = componentRegistry.getCommandNames();
+  coreCommandsArr.push("configure", "setDebug");
```

Comment:
Instead of using locally-defined strings for `"configure"` and `"setDebug"`, take [the constants from `src/core/injectExecuteCommand.js`](https://github.com/adobe/alloy/blob/main/src/core/injectExecuteCommand.js#L24), put them into a shared constants file [in `src/constants/`](https://github.com/adobe/alloy/blob/main/src/constants), and then use those new constants both here and in `injectExecuteCommand`. This will ensure consistency between command list definition and execution of those commands.